plugins/woocommerce/src/Admin/Features/NewProductManagementExperience.php

```diff
@@ -22,6 +23,24 @@ class NewProductManagementExperience {
 	 * Constructor
 	 */
 	public function __construct() {
+		$new_product_experience_param = 'new-product-experience-disabled';
+		if ( isset( $_GET[ $new_product_experience_param ] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			$url  = isset( $_SERVER['HTTPS'] ) && 'on' === $_SERVER['HTTPS'] ? 'https://' : 'http://';
```

Comment:
I think we can just use `remove_query_arg( 'new-product-experience-disabled', $url )` to remove the arg here.

https://developer.wordpress.org/reference/functions/remove_query_arg/